pixeltable/share/publish.py

```diff
@@ -68,7 +68,22 @@ def push_replica(
     # TODO: Use Pydantic for validation
     finalize_response = requests.post(PIXELTABLE_API_URL, json=finalize_request_json, headers=_api_headers())
     if finalize_response.status_code != 200:
-        raise excs.Error(f'Error finalizing snapshot: {finalize_response.text}')
+        # Check if this is a "table already replicated" error and provide actionable guidance
+        error_text = finalize_response.text
+        if 'already been replicated as' in error_text:
+            # Extract the existing table name from the error message
+            # Format: "That table has already been replicated as 'table_name'."
+            import re
```

Comment:
Imports should go at top of file unless they need to be in narrower scope.

Reply:
Fixed